plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/BaseMariaDbConnectorTest.java

```diff
@@ -254,18 +257,49 @@ public void testDeleteWithLike()
     @Override
     protected String errorMessageForCreateTableAsSelectNegativeDate(String date)
     {
-        return format("Failed to insert data: \\(conn=.*\\) Incorrect date value: '%s'.*", date);
+        return format("Failed to insert data: .* \\(conn=.*\\) Incorrect date value: '%s'.*", date);
     }
 
     @Override
     protected String errorMessageForInsertNegativeDate(String date)
     {
-        return format("Failed to insert data: \\(conn=.*\\) Incorrect date value: '%s'.*", date);
+        return format("Failed to insert data: .* \\(conn=.*\\) Incorrect date value: '%s'.*", date);
     }
 
     @Override
     protected String errorMessageForInsertIntoNotNullColumn(String columnName)
     {
-        return format("Failed to insert data: \\(conn=.*\\) Field '%s' doesn't have a default value", columnName);
+        return format("Failed to insert data: .* \\(conn=.*\\) Field '%s' doesn't have a default value", columnName);
+    }
+
+    @Override
+    protected void testDataMappingVerify(String trinoTypeName, String tableName, String sampleValueLiteral, String highValueLiteral)
```

Comment:
Add "--explicit-defaults-for-timestamp=1" in mariadb server start

Reply:
Do you know if there's a JDBC client option instead of server side?

Reply:
I did some investigation, seems the can NOT set in the JDBC, because it is Global Scope, and read only variable. Any ideas? Thanks!
```
explicit_defaults_for_timestamp
Description: This option causes [CREATE TABLE](https://mariadb.com/kb/en/create-table/) to create all [TIMESTAMP](https://mariadb.com/kb/en/timestamp/) columns as [NULL](https://mariadb.com/kb/en/null-values/) with the DEFAULT NULL attribute, Without this option, TIMESTAMP columns are NOT NULL and have implicit DEFAULT clauses.
Commandline: --explicit-defaults-for-timestamp=[={0|1}]
Scope: Global
Dynamic: No
Data Type: bolean
Default Value: OFF
```

```
MariaDB [tpch]> SET SESSION explicit_defaults_for_timestamp = 'ON';
ERROR 1238 (HY000): Variable 'explicit_defaults_for_timestamp' is a read only variable
```